classes/02class/exercises/c02-network03/jaysues/ANSWER.md

```diff
@@ -0,0 +1,72 @@
+# c02-network03
+
+## Commands Execution Output
+
+- Commands for creating a new VPC:
+```
+First we need to create the VPC:
+aws ec2 create-vpc --instance-tenancy default --cidr-block 192.168.0.0/16 -no-amazon-provided-ipv6-cidr-block 
```

Comment:
You could have defined the tags using "--tag-specifications" and providing the list
Also the region is missing where vpc need be created

Reply:
https://docs.aws.amazon.com/cli/latest/reference/ec2/create-vpc.html